apps/staking/src/app/app-wrapper.tsx

```diff
@@ -242,6 +243,8 @@ export function AppWrapper({ children }: PropsWithChildren) {
         onConnectClick={handleWalletConnect}
         onClose={closeSelectWallet}
       />
+
+      <Toaster />
```

Comment:
remove from here. put back to app-providers

Reply:
Done